linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/account_password_selinux_faillock_dir/tests/wrong_value.fail.sh

```diff
@@ -3,12 +3,8 @@
 # platform = multi_platform_all
 
 truncate -s 0 /etc/security/faillock.conf
-echo "dir=/var/log/faillock" >  /etc/security/faillock.conf
+echo "dir=/var/log/faillock" > /etc/security/faillock.conf
 
 mkdir /var/log/faillock
-
-semanage fcontext -a -t dummy_t "/var/log/faillock(/.*)?"
-
+semanage fcontext -a -t tmp_t "/var/log/faillock(/.*)?"
 restorecon -R -v "/var/log/faillock"
-
-
```

Comment:
While I agree not to exclusively remove white spaces in large or complex PRs, I advocate that we should remove even minor issues in files we are already editing. It is annoying to keep unnecessary spaces in a file that we are already modifying, especially if only short lines are affected. It's also very unlikely that a new PR will be created in the future just to remove unnecessary spaces, tending to keep those extra spaces forever if not removed when we have the opportunity. So, I use the good sense when I believe this will make the file quality better without significantly impact in the review. In this specific file it was changed the `fcontext`. And in the other two test scenario scripts the extra space was also removed to keep them aligned.

Reply:
You don't have to remove them from this PR but keep it in mind for the future.